manifests/00-crd-extension-console-link.yaml

```diff
@@ -66,3 +66,12 @@ spec:
                 imageURL:
                   type: string
                   description: The URL for the icon used in front of the link in the application menu.  The URL must be an HTTPS URL or a Data URI. The image should be square and will be shown at 24x24 pixels.
+            namespaceDashboard:
+              type: object
+              description: Object that holds information about namespaces in which the dashboard link should appear, and it is applicable only when location is set to NamespaceDashboard. If not specified, the link will appear in all namespaces.
+              required:
+                - namespaces
+              properties:
+                namespaces:
+                  type: array
```

Comment:
I believe need can use `items` to specify it's an array of string.

https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/#specifying-a-structural-schema

Reply:
```suggestion
                  type: array
                  items:
                    type: string
```